apps/daimo-mobile/src/model/account.ts

```diff
@@ -171,54 +196,15 @@ export function parseAccount(accountJSON?: string): Account | null {
   const model = JSON.parse(accountJSON) as StoredModel;
 
   // Migrations
-  // Delete V1 testnet account. Re-onboard, ask for notifications permisison.
-  if (model.storageVersion === 1) return null;
-
-  if (model.storageVersion === 2) {
-    const a = model as AccountV2;
-    return {
-      enclaveKeyName: a.enclaveKeyName,
-      name: a.name,
-      address: getAddress(a.address),
-
-      lastBalance: BigInt(a.lastBalance),
-      lastBlock: 0,
-      lastBlockTimestamp: a.lastBlockTimestamp,
-      lastFinalizedBlock: 0,
-
-      recentTransfers: [],
-      trackedRequests: [],
-      namedAccounts: [],
-
-      pushToken: a.pushToken,
-    };
-  }
-
-  if (model.storageVersion === 3) {
-    const a = model as AccountV3;
-    return {
-      enclaveKeyName: a.enclaveKeyName,
-      name: a.name,
-      address: getAddress(a.address),
-
-      lastBalance: BigInt(a.lastBalance),
-      lastBlock: a.lastBlock,
-      lastBlockTimestamp: a.lastBlockTimestamp,
-      lastFinalizedBlock: a.lastFinalizedBlock,
-
-      recentTransfers: a.recentTransfers,
-      trackedRequests: [],
-      namedAccounts: a.namedAccounts,
-
-      pushToken: a.pushToken,
-    };
-  }
+  // Delete V1-4 testnet accounts. Re-onboard to accounts with multi-key support.
```

Comment:
was skeptical, but i'm convinced now that it's good if & only if we ship concurrently with a much faster onboarding experience.

gives our testers a nice surprise